truss-chains/truss_chains/public_api.py

```diff
@@ -122,6 +122,19 @@ def __init_with_arg_check__(self, *args, **kwargs):
             cls.__init__ = __init_with_arg_check__  # type: ignore[method-assign]
 
 
+class ModelBase(definitions.ABCChainlet):
```

Comment:
This new superclass allows us to change behavior slightly between traditional chains and solo models being deployed using the chains framework. For now, the main difference is the `is_entrypoint` below, but can imagine this growing over time